eng/mgmt/mgmtmetadata/deviceprovisioningservices_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+Installing AutoRest version: v2
```

Comment:
was this file automatically generated?

Reply:
Yup. It allows us to track and verify generated code.